linode/manager
# refactor: Remove Cypress `cy.visitWithLogin()` account mocking, improve DBaaS mocks

## Description 📝
This is related to PR #9715 from yesterday -- we got the tests passing by mocking the `databases` feature flag to `true`, when the root cause of the failures was actually that the API `/account` request was getting mocked, and the mocked response did not include "Managed Databases" in the list of capabilities.

This PR makes a handful of changes to address this and restore the DBaaS tests to a state that's more useful for their team.

## Major Changes 🔄
- Adds `Cloud Firewall`, `Vlans`, `Machine Images`, and `Managed Databases` to the Account factory's `capabilities` property
- Removes implicit account mocking behavior from `cy.visitWithLogin()` -- this was added a while ago to get around performance issues with the API `/account` endpoint. I think the performance issues with that endpoint have been improved if not resolved, and opted to remove this behavior altogether due to the confusion caused
- Explicitly mock the account request in the DBaaS tests to ensure that the tests are running with the `Managed Databases` capability available, regardless of the state of the test accounts
- Remove the mocks related to the `databases` feature flag so that the DBaaS team can run the tests against environments where the flag is and is not enabled

## How to test 🧪
We can rely on automated tests to ensure that everything continues running and passing as expected. I want to pay particular attention to the performance of these test runs to ensure that removing the implicit account mocking didn't impact performance in a significant way.
